cmd/podman/system/service.go

```diff
@@ -39,7 +39,8 @@ Enable a listening service for API access to Podman commands.
 	}
 
 	srvArgs = struct {
-		Timeout int64
+		Timeout     int64
+		corsHeaders string
```

Comment:
I think this needs to be Capitalized (first letter, at least) so the value can be set by the CLI package

Reply:
Right, good catch